lib/aggregators/stacks-address.ts

```diff
@@ -44,35 +46,42 @@ class StacksAddress extends Aggregator {
     const address = c32check.c32ToB58(addr);
     const token = 'STACKS';
 
-    const [{ tokens }, history, status, balance, vestingTotal] = await Promise.all([
+    const [{ tokens }, history, status, balance, Vesting, cumulativeVestedAtBlocks, tokensGranted] = await Promise.all([
       network.getAccountTokens(address),
       this.getHistory(address),
       network.getAccountStatus(address, token),
       network.getAccountBalance(address, token),
-      getVestingTotalForAddress(addr),
+      getVestingForAddress(address),
+      this.getCumulativeVestedAtBlocks(address),
+      getTokensGrantedInHardFork(address),
     ]);
 
     let unlockInfo = {};
-    if (vestingTotal && vestingTotal > 0) {
+    if (Vesting.vestingTotal && Vesting.vestingTotal > 0) {
       unlockInfo = {
-        formattedUnlockTotal: accounting.formatNumber(vestingTotal * 10e-7),
-        unlockTotalStacks: stacksValue(vestingTotal),
-        unlockTotal: vestingTotal,
+        formattedUnlockTotal: accounting.formatNumber(Vesting.vestingTotal * 10e-7),
```

Comment:
Dangerous floating point operation here. Should use BigInt w/ polyfill or equivalent

Reply:
Wouldn't this indicate we need to refactor every single place that converts microstacks to Stacks?

Reply:
Not a blocker for this PR